Files/Views/LayoutModes/GenericFileBrowser.xaml

```diff
@@ -684,8 +684,12 @@
                     x:Name="PinItemToStart"
                     x:Uid="PinItemToStart"
                     x:Load="False"
-                    Click="PinItemToStart_Click"
                     Text="Pin to the Start Menu">
```

Comment:
Can't you use the command event?

Reply:
To pass in `EventArgs` you need to do it the way it's done now.

Reply:
@d2dyno1 Would setting the `CommandParameter` help?

Reply:
nope